### ISSUE TITLE
Product data model

### ISSUE THREAD

**Issue by jswk** [cyfronet-fid/sat4envi#33]
A data model to accomodate Sentinel and geostationary products.

The nomenclature has to shift. Product -> ProductType. Granule -> Product. 

**Geostationary.** Product will have fields: granuleUrl, layerName (until it isn't moved to per GS-instance corresponding item).

Collections covered in this issue: ProductTypes, Products, SentinelProducts, Granules, GranuleStatuses.

See: https://docs.cyfronet.pl/pages/viewpage.action?pageId=46335820 for DB diagram.

Blocks: #34.
Related: #47.

**Comment by kmarszalek** [comment]
extract sentinel aspect to new ticket.
refactor granules and products,

**Comment by kmarszalek** [comment]
Product -> ProductType
Granule -> Product
backend done.
changed API and refactored only call to it from s4e-web @michal-szostak
